pkg/trace/traceutil/span.go

```diff
@@ -132,3 +132,11 @@ func GetMetaStruct(s *pb.Span, key string) (interface{}, bool) {
 	}
 	return nil, false
 }
+
+// GetMetric gets the metadata value in the span Metrics map.
+func GetMetric(s *pb.Span, key string) apmsampling.SamplingMechanism {
```

Comment:
I am using that ```
if traceutil.GetMetric(span, sampler.KeySpanSamplingMechanism)
```

I can't use sampler.KeySamplingMechanism, it will cause import cycle

Reply:
nvm, I saw